includes/class-comment.php

```diff
@@ -715,4 +716,27 @@ public static function pre_comment_approved( $approved, $commentdata ) {
 
 		return $approved;
 	}
+
+	/**
+	 * Filters the comment count to exclude ActivityPub comment types.
+	 *
+	 * @param int|null $new_count The new comment count. Default null.
+	 * @param int      $old_count The old comment count.
+	 * @param int      $post_id   Post ID.
+	 *
+	 * @return int|null The updated comment count, or null to use the default query.
+	 */
+	public static function pre_wp_update_comment_count_now( $new_count, $old_count, $post_id ) {
+		if ( null === $new_count ) {
+			global $wpdb;
+
+			$excluded_types = self::get_comment_type_names();
+
+			// phpcs:ignore WordPress.DB
+			$new_count = (int) $wpdb->get_var( $wpdb->prepare( "SELECT COUNT(*) FROM $wpdb->comments WHERE comment_post_ID = %d AND comment_approved = '1' AND comment_type NOT IN ('" . implode( "','", $excluded_types ) . "')", $post_id ) );
```

Comment:
why not only search for "comment" instead?

Reply:
Pingbacks and trackbacks are part of the count, too. And we don't know what other custom comment types might exist

Reply:
good time to fix that too ;)